### ISSUE TITLE
Don't produce well-formed XML

### ISSUE THREAD

**Issue by 7daysofrain** [mjeanroy/gulp-header-comment#164]
When using it in XML files (like a SVG spritesheet in my case) the comment is written before the xml declaration. This produces invalid XML.

**Comment by mjeanroy** [comment]
Hi @7daysofrain,
Thanks for the report.
Version 0.5.0 (published few minutes ago) should fix this issue.
Feel free to comment if it does not work for you.

**Comment by 7daysofrain** [comment]
Works perfect, thanks!

**Comment by 7daysofrain** [comment]
Sorry, it has a bug that i just already found. The plugin messes around with the capitalization of the SVGs paths and the icons gets broken. For example it changes the path:
```
<path d="M122 98h14v12a9.56 9.56 0 0 1-7 9.5c-8.28 2.26-25 3.73-25-23.5 0-29.5 26.56-25.9 30.5-18.5M80 72v50M88 108H54l26-36" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="8"/>
```

into

```
<path d="m122 98h14v12a9.56 9.56 0 0 1-7 9.5c-8.28 2.26-25 3.73-25-23.5 0-29.5 26.56-25.9 30.5-18.5m80 72v50m88 108h54l26-36" fill="none" stroke="currentcolor" stroke-linecap="round" stroke-linejoin="round" stroke-width="8"></path>
```
(notice the change of 'M' into 'm' and 'H' into 'h'

Thanks

**Comment by mjeanroy** [comment]
Sorry, litte mistake, now fixed with 0.6.1 just published few minutes ago.
